spec/lib/myfinance/resources/receivable_account_spec.rb

```diff
@@ -161,4 +230,212 @@
       end
     end
   end
+
+  describe "#create_as_recurrent", vcr: true do
+    let(:params) { { due_date: "2015-10-20", amount: 150.99, create_as_recurrent: "monthly" } }
+    subject { client.receivable_accounts.create_as_recurrent(entity_id, params) }
+
+    context "with success" do
+      it "creates a new object" do
+        expect(subject.id).to eq(1298917)
+        expect(subject.due_date).to eq(Date.new(2015, 10, 20))
+        expect(subject.entity_id).to eq(entity_id)
+        expect(subject.status).to eq(1)
+        expect(subject.status_name).to eq("unreceived")
+        expect(subject.occurred_at).to be_nil
+        expect(subject.amount).to eq(150.99)
+        expect(subject.ticket_amount).to be_nil
+        expect(subject.interest_amount).to be_nil
+        expect(subject.discount_amount).to be_nil
+        expect(subject.total_amount).to be_nil
+        expect(subject.description).to be_nil
+        expect(subject.document).to be_nil
+        expect(subject.document_emission_date).to be_nil
+        expect(subject.observation).to be_nil
+        expect(subject.remind).to be_falsy
+        expect(subject.reminded_at).to be_nil
+        expect(subject.income_tax_relevant).to be_falsy
+        expect(subject.category_id).to be_nil
+        expect(subject.classification_center_id).to be_nil
+        expect(subject.expected_deposit_account_id).to be_nil
+        expect(subject.recurrence_id).to eq(8831)
+        expect(subject.person_id).to be_nil
+        expect(subject.created_at).to eq(DateTime.parse("2015-10-20T15:10:44-02:00"))
+        expect(subject.updated_at).to eq(DateTime.parse("2015-10-20T15:10:45-02:00"))
```

Comment:
Do not use `DateTime.parse` without zone. Use one of `Time.zone.parse`, `DateTime.parse.current`, `DateTime.parse.in_time_zone`, `DateTime.parse.utc`, `DateTime.parse.getlocal`, `DateTime.parse.iso8601`, `DateTime.parse.jisx0301`, `DateTime.parse.rfc3339`, `DateTime.parse.to_i`, `DateTime.parse.to_f` instead.